cmd/gitrepo/unregister.go

```diff
@@ -19,20 +19,8 @@ func unregister() *cobra.Command {
 		Long:  "unregister a previously registered configuration",
 		RunE: func(cmd *cobra.Command, args []string) (err error) {
 			repoAlias = args[0]
-			cfgFile, err := gitrepo.GetRepoCfgPath()
-			if err != nil {
-				return fmt.Errorf("%v: %w", gitrepo.ErrLoadCfgFile, err)
-			}
-			file, err := os.OpenFile(cfgFile, os.O_RDWR|os.O_CREATE, 0666)
-			if err != nil {
-				return fmt.Errorf("%v: %w", gitrepo.ErrLoadCfgFile, err)
-			}
-			defer func(f *os.File) {
-				err := f.Close()
-				if err != nil {
-					fmt.Printf("failed to close config file, error: %v\n", err)
-				}
-			}(file)
+			file, err := gitrepo.ReadDefaultConfig()
```

Comment:
Need to check the err value first

Reply:
It is being checked right?
```go
repoAlias = args[0]
			file, err := gitrepo.ReadDefaultConfig()
			if err != nil {
				return err
			}
			defer argocdio.Close(file)
			repoCtxCfg, err := gitrepo.LoadRepoCfg(file)
			if err != nil {
				return fmt.Errorf("%v: %w", gitrepo.ErrLoadCfgFile, err)
			}
			if len(repoCtxCfg.Repos) == 0 {
				fmt.Println("no repositories registered")
				return
			}
```

Reply:
We read the config and check the error, `argocdio.Close()` doesn't return an error, after loading the config too we check the error.

Reply:
It's weird, I was seeing a different version of the code. Perhaps a caching problem. Looks ok now after a refresh.